.github/workflows/chromatic.yml

```
@@ -43,7 +43,7 @@ jobs:
 
       - name: Chromatic - Dependency PR
         uses: chromaui/action@v1
-        if: ${{ github.event_name == 'pull_request' && contains(github.event.pull_request.labels.*.name, 'dependencies') }}
+        if: ${{ github.event_name == 'pull_request' && (contains(github.event.pull_request.labels.*.name, 'dependencies') || github.event.pull_request.user.login == 'dependabot') }}
```

Is it possible that a PR could be opened by a different user login than 'dependabot' and be given the label `dependencies`?

If not then can we simplify this to:

```suggestion
        if: ${{ github.event_name == 'pull_request' && github.event.pull_request.user.login == 'dependabot' }}
```

?

Same for above

---

I was thinking that if you kept both conditions then people could still run a full chromatic test on manual dependency updates by adding the Dependency label